charts/gitea/templates/gitea/init.yaml

```diff
@@ -16,6 +16,8 @@ stringData:
     # END: initPreScript
     {{- end }}
 
+    chmod 777 /data
```

Comment:
How come chmod 777 is necessary? Sounds like a bad idea?

Reply:
I deem this a gitea bug for now, 777 is fine until that fix comes back